source/whats-new.txt

```diff
@@ -39,6 +41,19 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.0:
+
+What's New in 6.0
+-----------------
+
+The {+driver-short+} v6.0 release includes the following features:
+
+- This release removes support for the ``addUser()`` helper command. Use the
```

Comment:
Suggestion:

Even though "helper command" might be a valid way to describe it, I think it might be easier to understand and not require additional explanation if called a "method". 
Also, I wonder whether "this release" might be repetitive since it's already introduced by the text preceding the list:

```suggestion
- Removes support for the ``addUser()`` method. Use the
```

This might apply to Breaking Changes update too.